index.bs

```diff
@@ -569,7 +579,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     : {{CollectedClientData/challenge}}
     :: The [=base64url encoding=] of |options|.{{MakeCredentialOptions/challenge}}
     : {{origin}}
-    :: The [=unicode serialization of an origin|unicode serialization=] of |rpId|
+    :: |effectiveDomain|
+    <!-- might need to have both {{origin}} and {{rpId}} in collectedClientData... ? -->
```

Comment:
ok, a couple items to note in reply here. first, setting `{{origin}}` to |effectiveDomain| is a mistake. mea culpa, sorry, dunno what I was thinking.  it should be set to the serialization of |callerOrigin|, yes?

>  is there any security impact to the rpId not being in the signed hash [of collectedClientData]

ans: there is none, because hash(RP ID) is the first 32 bytes of `authenticator data` -- i.e., we're signing it in any case.  :)